iot_verifier/src/poc.rs

```diff
@@ -743,6 +782,20 @@ mod tests {
         assert!(100 > (gmaps_dist.abs_diff(dist) as i32));
     }
 
+    #[test]
+    fn test_calc_cell_distance() {
+        // location 1 is 51.51231394840223, -0.2919014665284206 ( ealing, london)
+        // location 2 is 51.5649315958581, -0.10295780727096393 ( finsbury pk, london)
+        // google maps distance is ~14.32km
+        // converted co-ords to h3 index at resolution 15
+        let loc1 = 644459695463521437;
+        let loc2 = 644460986971331488;
+        let dist: i32 = calc_cell_distance(loc1, loc2).unwrap() as i32;
+        // verify the calculated cell distance is more than min cell distance
```

Comment:
we're missing a verification step. Is this what you meant?

```suggestion
        verify_witness_cell_distance(Some(loc1), Some(loc2)).unwrap();
```